presto-main/etc/catalog/thrift.properties

```diff
@@ -0,0 +1,10 @@
+#
+# WARNING
+# ^^^^^^^
+# This configuration file is for development only and should NOT be used be
+# used in production. For example configuration, see the Presto documentation.
+#
+
+connector.name=presto-thrift
+presto.thrift.client.addresses=127.0.0.1:1234
+
```

Comment:
Sorry, we actually don't want a extra blank line, just make sure file ends in a newline.